Parse email files attachments msg (#16676)
demisto/content: Packs/CommonScripts/ReleaseNotes/1_6_12.md

#### Scripts
##### ParseEmailFiles

>  - Fixed an issue that the script does not show the attachment name in the war room.

Reviewer: ```suggestion
 - Fixed an issue where the attachment name is not shown in the war room.
```
